mkdocs.yml

```diff
@@ -25,6 +25,10 @@ nav:
   - Cluster Integration: cluster-integration.md
   - Apply/Diff: apply-and-diff.md
   - Architecture: architecture.md
+  - Tutorials:
+    - tutorials/how-to-navigate-k8s-resources-and-edit-the-source-code.md
```

Comment:
how does this show in the menu? With the document title?

Reply:
Yes, I'm working on it.

Reply:
Resolved